### PR TITLE
Add styling for attribution credit

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

**Comment by discodavey** [comment]
Waiting for sign off from Chris H so I can then add the json to the component.

**Comment by discodavey** [comment]
This is for the pattern library not journal side.

File: source/_patterns/01-molecules/components/carousel-item.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ properties:
   title:
```
Comment (NuclearRedeye) [inline source/_patterns/01-molecules/components/carousel-item.yaml:24]: I can't view this molecule standalone, just have a blank page. Should I be able to?

Edit: Actually I can't seem to get any of the Carousel molecules to display be selecting them from the `molecules->components` menu. Other molecules work, so I assume this should too?

**Review by NuclearRedeye — COMMENTED** [review]
Test locally, when I can load the pages it works well. But some items aren't working. Not sure if this is a regression, if so it should be fixed. If not, then we should raise another ticket to cover fixing those.

File: source/_patterns/01-molecules/components/carousel-item.yaml (hunk 1 above)
Comment (NuclearRedeye) [reply source/_patterns/01-molecules/components/carousel-item.yaml:24]: So I just tested this locally without your changes and it's still broken, so not a regression as a result of your change. I'll raise a separate issue to cover fixing them.

**Review by NuclearRedeye — APPROVED** [review]

**Pull request merged**